### ISSUE TITLE
getElement return DOMElement

### ISSUE THREAD

**Issue by funerr** [n1k0/casperjs#1099]
I recently wanted to do the following:
```var domElem = this.getElementInfo(".foo");```

I want to query ```domElem``` further on, like so:
```var domElemChild = domElem.QUERYFUNCTIONHERE(".bar");```

If my second statement was in a loop or if the selector was something like: ```*``` then it would be a real hassle to write the second statement nicely.

All I need is a function that returns the element, not an object. Then I can leverage the scope attached to the Element, and continue to operate on it, like so:

    var domElem = this.getElement(".foo");
    var domElemChild = domElem.querySelector(".bar"); // we can use the native DOM api on this object

This fixes this, because the function I propose (```getElement/s```) returns an Element and not a detached Object with no context.

Reference: 
http://stackoverflow.com/questions/27115108/query-a-getelementinfo-object-in-casperjs

Notes:
- One aesthetic improvement could be to make an alias to the native DOM API like so:
```querySelector => getElement```
So the api will be more consistent.
- I didn't write comprehensive tests, someone should try to create a more "robust" test case.

**Comment by paazmaya** [comment]
Would you like to create a pull request for this feature?

**Comment by funerr** [comment]
@paazmaya, I did but it got closed. https://github.com/n1k0/casperjs/pull/1100

**Comment by paazmaya** [comment]
Ouh.. did not see that.

What was the issue with using `findOne` and `findAll` as oppose to the methods you are adding?
http://docs.casperjs.org/en/latest/modules/clientutils.html#findall

**Comment by funerr** [comment]
@paazmaya, first off, it was hard to find the clinetUtils at first, as I thought that it must be in the casper module. I thought that the syntaxtic "sugar" as mickaelandrieu called it was more than just that, it was a missing piece of the casper module. 
I thought that in order to use these methods I would have to bind the scope of the element to them from the outside, which sounded ridiculous. 

In summary, I wanted to further query the DOM/Peices of the DOM without haing to use a different module. but just have the necessary functions in the caper module - leveraging the scope that is bound to them.

**Comment by mickaelandrieu** [comment]
Sorry, but as the feature is already "in" casperjs this issue still stay closed.

I'm opposite to more sugar in casperjs /c @n1k0
